frontend/src/features/properties/list/PropertyListView.test.tsx

```diff
@@ -1,33 +1,192 @@
 import PropertyListView from './PropertyListView';
 import React from 'react';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+import { render } from '@testing-library/react';
 import renderer from 'react-test-renderer';
 import configureMockStore from 'redux-mock-store';
 import thunk from 'redux-thunk';
 import { ILookupCode } from 'actions/lookupActions';
 import * as API from 'constants/API';
 import { Provider } from 'react-redux';
 import * as reducerTypes from 'constants/reducerTypes';
+import service from '../service';
+
+// Set all module functions to jest.fn
+jest.mock('../service');
+const mockedService = service as jest.Mocked<typeof service>;
```

Comment:
The approach for this test is to mock only the **service** class/helper. No need to mock axios here...